glide.yaml

```diff
@@ -7,3 +7,6 @@ import:
 - package: kubevirt.io/kubevirt
   version: v0.7.0
   repo:    https://github.com/kubevirt/kubevirt.git
+- package: kubevirt.io/qe-tools
+  version: 13fc8573324cd2af42c88043b5a7c6e03bf39f95
```

Comment:
I just copied this from kubevirt repo, I am not sure if this is correct for us here

Reply:
putting `version: v0.1.2` instead